app/tests/test_flask_run.py

```diff
@@ -1,8 +1,10 @@
-from main import app
+import typing as tp
 
+from tests.setup_tests import *
```

Comment:
wildcard imports is a very bad practice, don't do it. why you need it here?

Reply:
use `conftest.py` and then you don't need to import fixtures https://docs.pytest.org/en/latest/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files

Reply:
done